src/components/shared/select-dropdown.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {Text, View} from 'react-native';
+
+import FontAwesome from 'react-native-vector-icons/FontAwesome';
+import Dropdown from 'react-native-select-dropdown';
+
+const SelectDropdown = React.forwardRef(
```

Comment:
Why do we use forwardRef? Is this Required

Reply:
yes
for resetting to initial value